optics-core/src/Optics/AffineFold.hs

```diff
@@ -147,6 +147,8 @@ infixl 3 `afailing` -- Same as (<|>)
 -- >>> isn't _Just Nothing
 -- True
 --
+-- The negation of this operator is 'Optics.Core.Extra.is' from
+-- "Optics.Core.Extra".
 isn't :: Is k An_AffineFold => Optic' k is s a -> s -> Bool
 isn't k s = not (isJust (preview k s))
```

Comment:
This should be `isNothing`. Can you please change that?

Reply:
OK